pkg/proxy/ipvs/netlink_linux.go

```diff
@@ -146,21 +146,6 @@ func (h *netlinkHandle) ListBindAddress(devName string) ([]string, error) {
 
 // If filterDev is specified, the result will discard route of specified device and cut src from other routes.
 func (h *netlinkHandle) GetLocalAddresses(dev, filterDev string) (sets.String, error) {
-	chosenLinkIndex, filterLinkIndex := -1, -1
-	if dev != "" {
```

Comment:
Trying to understand the value of this refactor. Moving the `dev != ""` checks into the method actually makes the code harder to understand IMO.

Reply:
um,  At first I just wanted to eliminate duplicate code and error messages in the print, and in fact I have been tangled for a while. Since you mentioned it, I am a little hesitant now.....